ios/RNMagicScript.xcodeproj/xcshareddata/IDETemplateMacros.plist

```diff
@@ -4,7 +4,7 @@
 <dict>
 	<key>FILEHEADER</key>
 	<string>
-//  Copyright (c) 2019 Magic Leap, Inc. All Rights Reserved
+//  Copyright (c) 2020 Magic Leap, Inc. All Rights Reserved
```

Comment:
Should we include 2019 as well? -> `2019-2020`

Reply:
According to information from @kpiascik we should set this to `2019-2020` but we're waiting for final confirmation.

Reply:
Please use this:
`// Copyright (c) 2019-2020 Magic Leap, Inc. All Rights Reserved.`

Reply:
Find & replace for current files will be provided in separate PR.